bot/cogs/sudo.py

```diff
@@ -44,8 +42,9 @@ def get_uptime(self) -> str:
         else:
             return f"{hours} hr, {minutes} mins, and {seconds} secs"
 
+    @staticmethod
     async def is_owner(ctx: Context) -> t.Union[bool, None]:
```

Comment:
Rename this to a more meaningful `is_dev` and use pure `bool` return, simply return `False` on the failed check
```suggestion
    async def is_dev(ctx: Context) -> bool:
```